assets/js/components/Work/Tabs/Preservation.jsx

```diff
@@ -26,7 +26,7 @@ const WorkTabsPreservation = ({ work }) => {
                     <td>{fileset.role}</td>
                     <td>{metadata ? metadata.originalFilename : " "}</td>
                     <td className="notification is-warning">
-                      TODO: Need this exposed in GraphQL
+                      {metadata.sha256}
```

Comment:
I'm working on the issue that is specific to preservation tab metadata. Do you want to put this one back so i can make the changes on my file? If not, can you please remove className from this tag?